common.groovy

```diff
@@ -23,7 +23,7 @@ TEST_JOB_ROOT_NAME = 'workflow-test'
 
 defaults = [
   tmpPath: '/tmp/${JOB_NAME}/${BUILD_NUMBER}',
-  envFile: "${tmpPath}/env.properties",
+  envFile: "${defaults.tmpPath}/env.properties",
```

Comment:
believe we still need to pull `tmpPath` out of the map so `envFile` can ref it:
```
tmpPath = '/tmp/${JOB_NAME}/${BUILD_NUMBER}'
defaults = [
    envFile: "${tmpPath}/env.properties",
]
​```
(used https://groovyconsole.appspot.com/ to test... the current form here throws exception)